### PR TITLE
Feature/ci 410 training options improve view

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/Web/Sfa.Das.Sas.ApplicationServices/ApprenticeshipSearchService.cs [hunk 1]
```diff
@@ -2,7 +2,6 @@
 using Sfa.Das.Sas.ApplicationServices.Logging;
 using Sfa.Das.Sas.ApplicationServices.Models;
 using Sfa.Das.Sas.ApplicationServices.Settings;
-
```
Comment (davehowlett) [inline src/Web/Sfa.Das.Sas.ApplicationServices/ApprenticeshipSearchService.cs]: Resharper suggests reordering the using statements. The suggestion is:

```
namespace Sfa.Das.Sas.ApplicationServices
{
    using System.Collections.Generic;
    using Logging;
    using Models;
    using Settings;
    using SFA.DAS.NLog.Logger;
```

File: src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Services/Mapping/ForProviderMappingHelper.cs [hunk 2]
```diff
@@ -42,18 +42,75 @@ public void WhenGetCommaList()
...
-        [TestCase("block release", "BlockRelease")]
-        [TestCase("at your location", "100PercentEmployer")]
-        [TestCase("day release, block release", "DayRelease", "BlockRelease")]
-        [TestCase("day release, at your location", "DayRelease", "100PercentEmployer")]
-        [TestCase("block release, at your location", "BlockRelease", "100PercentEmployer")]
-        [TestCase("day release, block release", "BlockRelease", "DayRelease")]
-        [TestCase("day release, at your location", "100PercentEmployer", "DayRelease")]
-        [TestCase("block release, at your location", "100PercentEmployer", "BlockRelease")]
-        public void WhenGetDeliveryOptionTextt(string expected, params string[] input)
+        [TestCase(
```
Comment (davehowlett) [inline src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Services/Mapping/ForProviderMappingHelper.cs]: The test cases are a little difficult to read. Could they be rationalised into a testcasesource and written out using string interpolation?

File: src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/ProviderDetailsPage.cs [hunk 3]
```diff
@@ -146,7 +202,7 @@ public void ShouldShowAddressIfThereAreMoreDeliveryModesApartFromEmployerLocatio
                     Town = "Town"
                 },
                 LocationAddressLine = "Test location name, Address 1, Address 2",
-                DeliveryModes = new List<string> { "BlockRelease", "100PercentEmployer", "DayRelease" },
+                DeliveryModes = new List<string> {"100PercentEmployer", "DayRelease" },
```
Comment (davehowlett) [inline src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Views/Provider/ProviderDetailsPage.cs]: minor thing - Resharper suggests a space at the beginning

File: src/Web/Sfa.Das.Sas.ApplicationServices/ApprenticeshipSearchService.cs (hunk 1 above)
Comment (MarkFCain) [reply src/Web/Sfa.Das.Sas.ApplicationServices/ApprenticeshipSearchService.cs]: yeah, done that now.

File: src/Web/Sfa.Das.Sas.Web/Content/src/styles/screen.scss [hunk 4]
```diff
@@ -427,7 +427,7 @@ aside h2:first-child {
 /*custom styles to override defaults*/
 
 .standard-detail{
-    .column-full{
+    .column-full{cd
```
Comment (davehowlett) [inline src/Web/Sfa.Das.Sas.Web/Content/src/styles/screen.scss]: is this a typo?

File: src/Web/Sfa.Das.Sas.Web/Controllers/ProviderController.cs [hunk 5]
```diff
@@ -39,8 +39,9 @@ public sealed class ProviderController : Controller
         public async Task<ActionResult> StandardResults(StandardProviderSearchQuery criteria)
         {
             string postCodeUrl;
-            var response = await _mediator.SendAsync(criteria);
```
Comment (davehowlett) [inline src/Web/Sfa.Das.Sas.Web/Controllers/ProviderController.cs]: there are apparently trailing whitespaces here ?

Comment (davehowlett) [reply src/Web/Sfa.Das.Sas.Web/Controllers/ProviderController.cs]: not your change, but possible NullReferenceException if criteria is null

File: src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs [hunk 6]
```diff
@@ -1,14 +1,12 @@
...
     using System.Linq;
-
+    using ApplicationServices.Models;
     using Microsoft.Ajax.Utilities;
 
-    using Sfa.Das.Sas.ApplicationServices.Models;
-
     public static class ProviderMappingHelper
     {
         public static string GetPercentageText(double? level)
```
Comment (davehowlett) [inline src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs]: method can be simplified into a single return statement:
--
return level == null ? "no data available" : $"{level}%";
--

File: src/Web/Sfa.Das.Sas.Web/Content/src/styles/screen.scss (hunk 4 above)
Comment (MarkFCain) [reply src/Web/Sfa.Das.Sas.Web/Content/src/styles/screen.scss]: yep, not sure where that was from, other than thinking I was in a command shell...

**Pull request merged**